### PR TITLE
Aprisa

### PR DESCRIPTION
**Pull request by loopodoopo**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ X ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

#### Aprisa support

Support in Librenms for Aprisa microwave connections.

### CODE REVIEW

**Comment by loopodoopo** [comment]
I believe the pull request became a bit of a mess. Not sure if you are able to accept the request like this?

**Comment by loopodoopo** [comment]
I believe the pull request became a bit of a mess. Not sure if you are able to accept the request like this?

File: LibreNMS/OS/Aprisa.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+use LibreNMS\Device\WirelessSensor;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessPowerDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessRssiDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessSnrDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessErrorsDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessFrequencyDiscovery;
+use LibreNMS\OS;
+
+<<<<<<< HEAD
```
Comment (laf) [inline LibreNMS/OS/Aprisa.php]: You've got some merge conflicts indicators in this file you'll need to cleanup.

File: includes/definitions/aprisa.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+os: aprisa
+text: 'Aprisa XE'
+type: wireless
+icon: 4RF
```
Comment (laf) [inline includes/definitions/aprisa.yaml]: Please lowercase the icon name here + lower case the actual file.

Same for group below.

Comment (laf) [reply includes/definitions/aprisa.yaml]: You will also need to lower case the mib dir.

File: includes/definitions/aprisa.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+os: aprisa
+text: 'Aprisa XE'
```
Comment (laf) [inline includes/definitions/aprisa.yaml]: If this is to deal with the Aprisa range then please remove the XE. If it's specific to the XE then the os name needs to be updated to reflect that.

**Comment by loopodoopo** [comment]
Made a new pull request #9435

**Pull request closed without merging**